tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -1408,29 +1462,42 @@ def test_search_runs_returns_expected_results_with_large_experiment(self):
 
         run_results = self.store.search_runs([experiment_id], None, ViewType.ALL, max_results=100)
         assert len(run_results) == 100
-        assert set([run.info.run_id for run in run_results]).issubset(set(run_ids))
+        # runs are sorted by desc start_time
+        self.assertListEqual([run.info.run_id for run in run_results],
+                             list(reversed(run_ids[900:])))
 
     def test_search_runs_correctly_filters_large_data(self):
         experiment_id, _ = self._generate_large_data(1000)
 
         run_results = self.store.search_runs([experiment_id],
                                              "metrics.mkey_0 < 26 and metrics.mkey_0 > 5 ",
-                                             ViewType.ALL, max_results=1000)
+                                             ViewType.ALL, max_results=50)
         assert len(run_results) == 20
 
         run_results = self.store.search_runs([experiment_id],
                                              "metrics.mkey_0 < 26 and metrics.mkey_0 > 5 "
                                              "and tags.tkey_0 = 'tval_0' ",
-                                             ViewType.ALL, max_results=1000)
+                                             ViewType.ALL, max_results=10)
         assert len(run_results) == 2  # 20 runs between 9 and 26, 2 of which have a 0 tkey_0 value
 
         run_results = self.store.search_runs([experiment_id],
                                              "metrics.mkey_0 < 26 and metrics.mkey_0 > 5 "
                                              "and tags.tkey_0 = 'tval_0' "
                                              "and params.pkey_0 = 'pval_0'",
-                                             ViewType.ALL, max_results=1000)
+                                             ViewType.ALL, max_results=5)
         assert len(run_results) == 1  # 2 runs on previous request, 1 of which has a 0 pkey_0 value
 
+    def test_get_attribute_name(self):
```

Comment:
Can you merge this test with the pre-existing `test_get_attribute_name()` test that you introduced in the filter pushdown PR? This test is not specific to sqlite backends.